context-free.md

```diff
@@ -2,21 +2,28 @@
 
 ##  Who is this programming language for?
 
+Anyone who is interested in creating procedurally generated art.
 
 ## What is easy to do in this language? Why is it easy?
 
+Creating self similar images is very straight forward due to the way the rules call each other.
 
 ## What is hard to do in this language? Why is it hard?
 
+Controlling randomness seems a bit hard. It would be nice to do something other than just
```

Comment:
I would argue that controlling randomness is hard in any language (not just ContextFree) because pure randomness can't be controlled. Yes, it can take many trials and errors to find a random seed that creates the specific image in your mind, but why use randomness when you can deterministically build your image?